x-pack/plugins/security_solution/server/lists_integration/endpoint/validators/base_validator.ts

```diff
@@ -74,6 +77,22 @@ export class BaseValidator {
     }
   }
 
+  protected async validateHasEndpointExceptionsPrivileges(
+    privilege: keyof EndpointAuthz
+  ): Promise<void> {
+    const appFeaturesService = this.endpointAppContext.getAppFeaturesService();
+
+    const isEndpointExceptionsFeaturesEnabled = (await appFeaturesService).isEnabled(
+      AppFeatureSecurityKey.endpointExceptions
+    );
```

Comment:
Why is the `this.endpointAppContext.getAppFeaturesService()` an `async` function? The `AppFeaturesService` does not need to do any async logic, it is available synchronously.